grants_tagger/download_model.py

```diff
@@ -0,0 +1,40 @@
+import tarfile
+import os
+
+from tqdm import tqdm
+import requests
+
+MODELS = {
+    'disease_mesh': {
+        'url': 'https://github.com/wellcometrust/grants_tagger/releases/download/v0.1.3/disease_mesh_cnn-2021.03.1.tar.gz',
+        'path': 'disease_mesh_cnn-2021.03.1.tar.gz'
```

Comment:
Are you going to update this line every time there is a new model? That might be ok in the short run as we don't have many models, but in the long run it'd be ace if you had a `get_latest_version` function so that the dispatcher only has a "base" path `disease_mesh_cnn`. I think spacy has it for free, because their models are also released as pypi packages, but you could emulate that too.

So the user can either `grants_tagger download disease_mesh` and that will give you latest or `grants_tagger download disease_mesh_cnn-2021.03.1.tar.gz`

Reply:
I was actually thinking that the user will always download the model compatible with the version released. So when a new version is released in theory models would need to be retrained and this updated.